src/components/composite/WalletBalanceCard/WalletBalanceCard.tsx

```
@@ -132,22 +136,32 @@ export const WalletBalanceCard: FC<WalletBalanceCardProps> = ({
+                <BaseAlert
+                  variant={BaseAlertVariant.Error}
+                  description={error.message}
                 />
-              ))}
+              )}
+              {!error && (
+                <>
+                  {!isSuccess &&
+                    balanceGroups.length == 0 &&
+                    Array.from({ length: 8 }).map(() => (
+                      <BalanceCardSkeleton size={BalanceCardSize.SM} />
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)